src/components/messageFlowButton.vue

```diff
@@ -1,19 +1,21 @@
 <template>
-  <img
-    :src="messageFlow"
-    @click="$emit('click')"
+  <crown-button
     id="message-flow-button"
-    data-test="message-flow-button"
-    width="19"
-    height="20"
-    alt="Create a message flow"
+    aria-label="Create a message flow"
+    @click="$emit('click')"
   >
-
+    <img
+      :src="messageFlow"
+      alt=""
+    >
```

Comment:
For the delete button you  had additional properties:
```
aria-hidden="true"
data-prefix="fas"
data-icon="trash-alt"
class="svg-inline--fa fa-trash-alt fa-w-14"
```

Are these not required here? I think this should be consistent. I think common properties can be applied to the slot in crown button instead of specifying them individually on each `img`.

Reply:
The trash icon seems to use font-awesome. I wonder if it could be simplified.

Reply:
```
aria-hidden="true"
data-prefix="fas"
data-icon="trash-alt"
class="svg-inline--fa fa-trash-alt fa-w-14"
```

Those properties are just for the trash icon. I can't seem to remove them. They were originally in the svg file, but they aren't valid there.

Any thoughts on what to do with it?

Reply:
You could allow the crown button component to receive an optional array of key value pairs as a prop. These could then be iterated over to set any additional properties.

Reply:
Not sure how you'd inject those into the image on crownButton. Seems like a lot of work for 1 out of 6 icons. Maybe create the image in javascript and use v-html on the crownButton?

Reply:
I'm not sure I understand: what is the problem with leaving this as is?

Reply:
I'm fine with the slot to override the image for now. It would be nice if it was the same as the other svqs at some point.